taxonomies/food/ingredients.txt

```diff
@@ -80918,30 +80816,6 @@ ca: aroma de cranc
 es: aroma de cangrejo
 hr: okus rakova
 
-# description:en:Cancer pagurus, commonly known as the edible crab or brown crab, is a species of crab found in the North Sea, North Atlantic Ocean
-
-< en:crab
-en: edible crab, brown crab
```

Comment:
This was merge with crab because edible is defined as a stop word !

We should put brown crab as first entry !